### PR TITLE
Manual Way Added

### PR DESCRIPTION
**Pull request by kyupp**
-Need to verify if everything it's okay.

### CODE REVIEW

**Review by leanghok120 — CHANGES_REQUESTED** [review]
Since we already talked about how to install sddm and enabling it in the article, can you remove the prerequisite part and enabling sddm part, everything looks gud tho

**Comment by kyupp** [comment]
Fixed

File: articles/ricing_sddm.md [hunk 1]
```diff
@@ -62,7 +62,50 @@ After the script is finished, reboot your system and you should see a minimal di
...
+        - **Note:** You need to change "[folder location]", for example ```/usr/share/sddm/themes/where_is_my_sddm_theme_qt5```
+
+2. How to set "Where is my sddm theme" as the current sddm theme:
+  - **Note:** We are going to change the default file of sddm, it's not a good practice but it's the only place we found it.
+  - First we are going to open the directory where the sddm config file is, so execute this on your terminal:
+    - ```cd /usr/lib/sddm/sddm.conf.d/```
+    - Once we are in this directory just open the config file:
+      - ```sudo vim default.conf```
+      - Here you only need to change a line, so be careful and don't edit anything else.
+      - Inside vim you can use ```/Current``` to search for the line will be editing, so once you found it just change after the ```=``` and just add ```wheres_my_sddm_theme_qt5```
```
Comment (leanghok120) [inline articles/ricing_sddm.md]: The theme name is wrong

**Review by leanghok120 — APPROVED** [review]
LGTM!

**Pull request merged**